diff --git a/node_modules/@acala-network/chopsticks-core/dist/esm/wasm-executor/browser-worker.js b/dist/esm/wasm-executor/browser-worker.js
index 7007949a52dc3d4a824c035c2995203289722b13..5513e9158c43dc44ba9147d3954d1f0eb0e09a58 100644
--- a/node_modules/@acala-network/chopsticks-core/dist/esm/wasm-executor/browser-worker.js
+++ b/node_modules/@acala-network/chopsticks-core/dist/esm/wasm-executor/browser-worker.js
@@ -1,6 +1,6 @@
 import { wrap } from 'comlink';
 export const startWorker = async ()=>{
-    const worker = new Worker(new URL('browser-wasm-executor.js', import.meta.url), {
+    const worker = new Worker(new URL('@acala-network_chopsticks-core_wasm-executor_browser-wasm-executor.js', import.meta.url), {
         type: 'module',
         name: 'chopsticks-wasm-executor'
     });
